docs/publishing.md

```diff
@@ -0,0 +1,70 @@
+### Artifact distribution
+This project uses SDK Registry to distribute the binaries.
+
+1. Push an artifact:
+- Cut and push a new tag. It will publish a new artifact to `https://api.mapbox.com/downloads/v2/releases/maven`. The tag has to match the following pattern:
```

Comment:
nit: in regards to the link https://api.mapbox.com/downloads/v2/releases/maven

do these sites work? i'm using different access tokens trying to see the artifacts but they're not coming up. should we link to a working site?

Reply:
This is just an api base url, the actual url will be generated by gradle when the project is synced.